api/conf/schema.json

```diff
@@ -3217,21 +3217,21 @@
 				"minProperties": 1,
 				"properties": {
 					"body": {
-						"description": "new body for repsonse",
+						"description": "new body for response",
```

Comment:
I think we need to update the schema of APISIX too

@jbampton how did you find those problems? Any tools?

Reply:
Hey @membphis I use various tools like linters, spell checkers and also the find and replace function in my editor 👍